data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
+
+    [achievement]
+        id="HttT_S23_Lord_Bayar_of_the_Plains"
+        name= _ "Lord Bayar of the Plains"
+        name_completed= _ "Lord Bayar of the Plains"
+        description= _ "Confront and defeat Lord Bayar in <i>Test of the Clans</i>"
+        description_completed= _ "Bested Lord Bayar in <i>Test of the Clans</i>"
+        icon="units/human-loyalists/grand-knight/grand-knight-se-defend1.png~RC(magenta>black)~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/human-loyalists/grand-knight/grand-knight-se-defend2.png~RC(magenta>black)~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S23_Conquerer_of_the_Plains"
+        name= _ "Conquerer of the Plains"
+        name_completed= _ "Conquerer of the Plains"
+        description= _ "Defeat all Grand Knights in <i>Test of the Clans</i>"
+        description_completed= _ "Bested all Grand Knights in <i>Test of the Clans</i>"
+        icon="units/human-loyalists/grand-knight/grand-knight-se-defend1.png~RC(magenta>gold)~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/human-loyalists/grand-knight/grand-knight-se-defend2.png~RC(magenta>gold)~SCALE(72,72)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S24_Battle_for_Wesnoth"
+        name= _ "Battle for Wesnoth"
+        name_completed= _ "Battle for Wesnoth"
+        description= _ "Fight the final battle against the treacherous Dark Queen for the Throne of Wesnoth!"
+        description_completed= _ "Fought the final battle against the treacherous Dark Queen for the Throne of Wesnoth!"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/konrad-lord.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="data/campaigns/Heir_To_The_Throne/images/units/konrad-lord.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S24_Smite_Down_Evil_Konrad"
+        name= _ "Smite Down Evil with Konrad"
+        description= _ "Konrad dealt the final blow to the Dark Queen"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/konrad-lord.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S24_Smite_Down_Evil_Lisar"
+        name= _ "Smite Down Evil with Li'Sar"
+        description= _ "Li'Sar dealt the final blow to the Dark Queen"
+        icon="data/campaigns/Heir_To_The_Throne/images/units/human-princess.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S24_Smite_Down_Evil_Delfador"
+        name= _ "Smite Down Evil with Delfador"
```

Comment:
Heh, I'm impressed at the restraint you showed here by stopping yourself from making the name here a reference to your headcanon ship between the two of them... 😜